components-core/src/main/java/com/adyen/checkout/components/core/internal/data/api/AnalyticsMapper.kt

```diff
@@ -75,5 +75,24 @@ class AnalyticsMapper {
         private const val DROP_IN_COMPONENT = "dropin"
         private const val ANDROID_PLATFORM = "android"
         private const val ANDROID_CHANNEL = "android"
+
+        private var VERSION = BuildConfig.CHECKOUT_VERSION
+        private var PLATFORM = ANDROID_PLATFORM
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun overrideVersion(version: String) {
+            VERSION = version
+        }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun overridePlatform(platform: String) {
```

Comment:
I suggest making the argument an enum, with only 2 options: `REACT_NATIVE` and `FLUTTER`, wdyt?

Reply:
Yeah maybe that's better

Reply:
Not sure if this is better, but what if we merge these 2 methods and change the name to something like `overrideForCrossPlatform`, to make it even more obvious that this method shouldn't be used by anyone, and since our flutter/RN libs will always call both methods at the same time 🤔

Reply:
Good one, I like that idea